media/js/mozorg/home/takeover-2016.js

```diff
@@ -19,6 +19,18 @@
     var $optionLabels = $('label[class!="own-amount"]', $donateOptionsSection);
     var $options = $('input[type="radio"]', $donateOptionsSection);
 
+    var locale = $form.data('locale');
+    var currencyCode = $form.data('currency');
+    var numberFormat = new Intl.NumberFormat(locale, {
```

Comment:
Yeah, I haven't tested in older browsers but I imagine it would. Thinking I'll just wrap this in some feature detection and let it fall back to the inline text.